src/Fhir.Anonymizer.Core/Constants.cs

```diff
@@ -6,7 +6,7 @@ namespace Fhir.Anonymizer.Core
 {
     internal static class Constants
     {
-        internal const int DefaultPartitionedExecutionCount = 5;
+        internal const int DefaultPartitionedExecutionCount = 4;
```

Comment:
noticed that changed from 5 to 4, any reason?

Reply:
4 here is just a default value, I just choose 4 as default. Our tool set the count based on the logical cores. Thanks